lib/graphql_devise/resource_loader.rb

```diff
@@ -10,20 +10,20 @@ def initialize(resource, options = {}, routing = false)
     end
 
     def call(query, mutation)
-      mapping_name = @resource.to_s.underscore.tr('/', '_').to_sym
-
       # clean_options responds to all keys defined in GraphqlDevise::MountMethod::SUPPORTED_OPTIONS
       clean_options = GraphqlDevise::MountMethod::OptionSanitizer.new(@options).call!
 
-      return clean_options if GraphqlDevise.resource_mounted?(mapping_name) && @routing
+      model = @resource.is_a?(String) ? @resource.constantize : @resource
```

Comment:
Is this meant to support previous versions?

Reply:
Yes, we can't break support for string in the next version, but we can deprecate it as a prep for v1